### PR TITLE
fix: position of the custom network input

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

=Before=
image

=After=

Screenshot 2022-04-04 at 6 39 43 PM

Screenshot 2022-04-04 at 6 39 59 PM

Screenshot 2022-04-04 at 6 40 11 PM

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

### CODE REVIEW

File: src/components/header/modals/ModalNetwork.vue [hunk 1]
```diff
@@ -48,13 +48,15 @@
...
               </label>
               <!-- custom endpoint -->
-              <input
-                v-if="provider.key === endpointKey.CUSTOM && selNetwork === endpointKey.CUSTOM"
-                v-model="newEndpoint"
-                type="text"
-                placeholder="IP Address / Domain"
-                class="ip-input"
-              />
+              <div class="row--ip-input">
```
Comment (hoonsubin) [inline src/components/header/modals/ModalNetwork.vue:51]: Screen Shot 2022-04-04 at 11 22 21 AM

The width of the network button (`314px`) and the width of the network input field (`330px`) seems to be different.
@Kahonnohak is this intentional?

**Review by hoonsubin — APPROVED** [review]
LGTM

File: src/components/header/modals/ModalNetwork.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/header/modals/ModalNetwork.vue:51]: @hoonsubin @sirius651
I think [this](https://gyazo.com/219e1478b04b387ef9884fa3668be4b1) is the original design made by @Kahonnohak 
Let me fix the styling quickly.

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Review by sirius651 — APPROVED** [review]

**Comment by impelcrypto** [comment]
I'll merge this PR once I adjusted some styling issues.

File: src/components/header/modals/ModalNetwork.vue (hunk 1 above)
Comment (impelcrypto) [reply src/components/header/modals/ModalNetwork.vue]: Fixed.

Screenshot 2022-04-04 at 6 40 11 PM

Screenshot 2022-04-04 at 6 39 59 PM

Screenshot 2022-04-04 at 6 39 43 PM

**Comment by Kahonnohak** [comment]
Thanks and sorry for my absence

**Pull request merged**